staging/src/k8s.io/kms/internal/plugins/_mock/kms.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mock-kmsv2-provider
+  namespace: kube-system
+  labels:
+    tier: control-plane
+    component: mock-kmsv2-provider
+spec:
+  hostNetwork: true
```

Comment:
Is this required?

Reply:
We deploy the kms plugin as a static pod, so this is required. The kms plugin pod needs to start up before the CNI plugins are initialized.

This is the error log in kubelet for ref (with `hostNetwork: false`):
```bash
Sep 28 17:18:43 kms-control-plane kubelet[265]: E0928 17:18:43.305817     265 pod_workers.go:1300] "Error syncing pod, skipping" err="network is not ready: container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized" pod="kube-system/mock-kmsv2-provider-kms-control-plane" podUID="f4d3a22b9df2f76a09fb51308d0fbdb2"
```